src/Project/Classes/Comparator/ProductComparator.java

```diff
@@ -0,0 +1,11 @@
+package Project.Classes.Comparator;

```

Comment:
[Fix] pachetul de baza al oricarui proiect, sub src, dupa conventie, este dupa numele de domeniu al organizatiei, reversat (ex: ro.unibuc). in cazul nostru, putem folosi com.company, com.example, etc. In acel pachet se pun restul de pachete